hibernate-core/src/main/resources/org/hibernate/xsd/mapping/mapping-3.1.0.xsd

```diff
@@ -2277,12 +2277,14 @@
                     </xsd:annotation>
                     <xsd:attribute name="name" use="required" type="xsd:string"/>
                     <xsd:attribute name="type" use="required" type="xsd:string"/>
+                    <xsd:attribute name="resolver" type="xsd:string"/>
```

Comment:
I haven't thought about whether it is sensible to add anything to a versioned XSD. Maybe we should rather create a mapping-3.1.1.xsd? Wdyt @sebersole?

Reply:
As we discussed on the call, let's change to base this version on Hibernate version rather than JPA version

Reply:
After our team meeting, we decided to create a new mapping file per ORM major/minor version, so this change would have to go to the new file. @sebersole is going to commit these changes later today I hope, then you can rebase. After that, I think this is good to go.

Reply:
@beikov @sebersole any update on this?

Reply:
I've already started this on the 7.0 branch.  

I am getting ready to release 6.5, so I took your branch and and making the change on top of that.  I'll send a PR in a few